src/Uno.UWPSyncGenerator/Generator.cs

```diff
@@ -122,16 +121,28 @@ where Path.GetFileNameWithoutExtension(externalRedfs.Display).StartsWith("Window
 #endif
 			};
 
+			var compositionNamespaces = new[]
+			{
+				"Windows.UI.Composition",
+#if HAS_UNO_WINUI
+				"Microsoft.UI.Composition"
+#endif
+			};
+
 			var q = from asm in origins
 					where asm.Name == sourceAssembly
 					from targetType in GetNamespaceTypes(asm.Modules.First().GlobalNamespace)
 					where !SkipNamespace(targetType)
 					where targetType.DeclaredAccessibility == Accessibility.Public
 					where ((baseName == "Uno" || baseName == "Uno.Foundation") && !targetType.ContainingNamespace.ToString().StartsWith("Windows.UI.Xaml") && !targetType.ContainingNamespace.ToString().StartsWith("Microsoft.UI.Xaml"))
 					|| (
-						(baseName == "Uno.UI" || baseName == "Uno.UI.Dispatching" || baseName == "Uno.UI.Composition")
+						(baseName == "Uno.UI" || baseName == "Uno.UI.Dispatching")
```

Comment:
I'm not sure why `Uno.UI.Dispatching` is here. It doesn't seem to have any generated files. In fact, it doesn't have any public APIs.